Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/EditorModeOutline.azsl

```diff
@@ -16,44 +16,64 @@ partial ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
 
     //! Outline line color.
     float4 m_lineColor;
+
+    //! Outline style to use ()
+    uint m_outlineStyle;
 }
 
 //! Applies an outline effect to the entity mask of the specified line thickness.
-//! @note There are known issues with pimples due to z-fighting of coincident surfaces in the entity mask that will be addressed it LYN-10304.
-float Outliner(const float2 uv, const float lineThickness)
+//! @returns The outlines of both channels of the entity mask.
```

Comment:
What outline is in the first channel and what outline is in the second? Comment is incomplete

Reply:
Fixed 👍